public/index.js

```diff
@@ -214,7 +214,11 @@ function confirmAddrMempool (block) {
       var confirms = row.children('.addr-tx-confirms')
       confirms.attr('data-tx-block-height', block.height)
       row.removeAttr('data-addr-tx-pending')
-      row.children('.addr-tx-time').html(formatTxDate(block.time, false))
+      //row.children('.addr-tx-time').html(formatTxDate(block.time, false))
+      var addrTime = row.children('.addr-tx-time').children('span')
+      addrTime.removeAttr('data-addr-tx-unconfirmed') // tell the address controller how to interpret the following time
+      addrTime.attr('data-addr-tx-time', block.time) // the address controller will handle it from here
+      addrTime.textContent = block.formatted_time
```

Comment:
At this point I realized it's just going to keep getting hairier.  Combining the mempool rows logic into address controller in addresses_controller.js is preferable.